packages/react-components/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -14,6 +14,8 @@ export const buttonClassNames: SlotClassNames<ButtonSlots> = {
  */
 export const buttonClassName = buttonClassNames.root;
 
+const iconSpacingVar = '--fui-Button__icon--spacing';
```

Comment:
This CSS var name is consistent with the way we name CSS classes on slots but is different from the way we name our token CSS vars. Should the name be changed to align with token CSS vars?

Reply:
I used the approach we have in `useSliderStyles` since that's the only other place I'm aware that we're using tokens.

Reply:
I've created a follow-up issue to figure out our naming convention for component-level css variables, so we don't block this PR on that. It can be found at #23095.